oras.yaml

```diff
@@ -0,0 +1,40 @@
+package:
+  name: oras
+  version: 0.16.0

```

Comment:
[ORAS CLI v1.0.0-rc.1](https://github.com/oras-project/oras/releases/tag/v1.0.0-rc.1) is the latest version. Do you consider using it here?

Reply:
I guess I can change to rc release if no one oppose to this

Reply:
I assume, you can go for it @tuananh

Reply:
> [ORAS CLI v1.0.0-rc.1](https://github.com/oras-project/oras/releases/tag/v1.0.0-rc.1) is the latest version. Do you consider using it here?

Though this a pre release?  I wonder if we should keep to released versions.

Reply:
Lets do 0.16.0 and wait for 1.0.0